lib/main.dart

```diff
@@ -183,18 +218,31 @@ class _MyHomePageState extends State<MyHomePage> {
           child: Column(
             mainAxisAlignment: MainAxisAlignment.center,
             children: <Widget>[
-              Text(
-                AppLocalizations.of(context)!.tapToFindALanguagePartner,
-                style: Theme.of(context).textTheme.headlineSmall,
-                textAlign: TextAlign.center,
-              ),
-              const SizedBox(
-                height: 32,
+              Expanded(
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.end,
+                  children: [
+                    Text(
+                      callToActionText(timerDuration),
+                      style: Theme.of(context).textTheme.headlineSmall,
+                      textAlign: TextAlign.center,
+                      overflow: TextOverflow.visible,
+                    ),
+                  ],
+                ),
               ),
-              FloatingActionButton.large(
-                onPressed: _launchUrl,
-                tooltip: AppLocalizations.of(context)!.call,
-                child: const Icon(Icons.phone),
+              const SizedBox(height: 32),
+              Expanded(
```

Comment:
I think this double Expanded solution only works to keep our button "fixed" because they have equal weight.